crates/sui-core/src/consensus_types/consensus_output_api.rs

```diff
@@ -109,7 +109,8 @@ impl ConsensusOutputAPI for consensus_core::CommittedSubDag {
             Some(
                 self.reputation_scores
```

Comment:
I think you need to add a sort here, IIUC `update_low_scoring_authorities` is expected it to be sorted and you no longer sort it before sending it to Sui

Reply:
Good point! Propagating the sorted scores to Sui now.